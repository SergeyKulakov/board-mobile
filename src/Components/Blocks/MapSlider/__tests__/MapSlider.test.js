import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { MapSlider as Component } from '../index'

describe('MapSlider', () => {
  const defaultProps = {
    data: [],
    renderItem: () => null,
    extraData: [],
    onChangeIndex: () => null,
    onCreateRef: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
