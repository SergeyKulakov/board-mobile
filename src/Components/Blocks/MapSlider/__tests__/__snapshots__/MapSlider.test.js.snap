// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSlider render snapshot 1`] = `
<Styled(View)>
  <_default
    data={Array []}
    extraData={Array []}
    itemWidth={1000}
    onBeforeSnapToItem={[Function]}
    renderItem={[Function]}
    sliderWidth={750}
    style={
      Object {
        "flex": 1,
      }
    }
  />
</Styled(View)>
`;
